DataConnectors/OracleWebLogicServer/Connector_OracleWebLogicServer_agent.json

```diff
@@ -0,0 +1,160 @@
+{
+    "id": "OracleWebLogicServer",
+    "title": "Oracle WebLogic Server",
+    "publisher": "Oracle",
+    "descriptionMarkdown": "OracleWebLogicServer data connector provides the capability to ingest [OracleWebLogicServer](https://docs.oracle.com/en/middleware/standalone/weblogic-server/index.html) events into Azure Sentinel. Refer to [OracleWebLogicServer documentation](https://docs.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/index.html) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **OracleWebLogicServerEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-OracleWebLogicServer-parser)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "OracleWebLogicServer_CL",
+            "baseQuery": "OracleWebLogicServerEvent"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Devices",
+            "query": "OracleWebLogicServerEvent\n | summarize count() by DvcHostname\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "OracleWebLogicServer_CL",
+            "lastDataReceivedQuery": "OracleWebLogicServerEvent\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "OracleWebLogicServerEvent\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
```

Comment:
Status should be 2. Please update the status.

Thanks,
Aditi Maheshwari

Reply:
updated